watertap/unit_models/pressure_exchanger.py

```diff
@@ -201,24 +232,46 @@ def build(self):
 
         units_meta = self.config.property_package.get_metadata().get_derived_units
 
-        self.efficiency_pressure_exchanger = Var(
-            self.flowsheet().config.time,
-            initialize=0.95,
-            bounds=(1e-6, 1),
-            domain=NonNegativeReals,
-            units=pyunits.dimensionless,
-            doc="Pressure exchanger efficiency",
-        )
+        if self.config.pressure_exchange_calculation is PressureExchangeType.efficiency:
+            self.efficiency_pressure_exchanger = Var(
```

Comment:
I'm not sure that efficiency should be excluded in the new configuration. I would think we should keep it in and just calculate instead of fixing it.

Reply:
I agree

Reply:
Agreed! We should add an constraint to calc efficiency, which also means ,this constraint can be used to calculate one of he parameters in the new model.